### PR TITLE
[FAT-1540] Implement API Karate tests - MARC transformation options

### PR DESCRIPTION
**Pull request by aleksandr2110**
Purpose

Karate implementation is created for the following endpoints

GET /central-servers/{centralServerId}/marc-transformation-options
POST /central-servers/{centralServerId}/marc-transformation-options
PUT /central-servers/{centralServerId}/marc-transformation-options
DELETE /central-servers/{centralServerId}/marc-transformation-options

### CODE REVIEW

File: mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/marc-transformation-options-settings.feature [hunk 1]
```diff
@@ -14,23 +14,104 @@ Feature: MARC transformation options settings
...
+    * print 'Prepare central servers'
+    * callonce read(featuresPath + 'central-server.feature@create')
+    * def centralServer1 = response.centralServers[0]
+    * def centralServer2 = response.centralServers[1]
+    * def notExistedCentralServerId1 = globalCentralServerId1
 
-  @Undefined
+  @create
   Scenario: Create MARC transformation options settings
+    * configure headers = headersUser
```
Comment (oliinyko) [inline mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/marc-transformation-options-settings.feature:27]: headers are already set in background section

Comment (aleksandr2110) [reply mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/marc-transformation-options-settings.feature:27]: Deleted everywhere

File: mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/marc-transformation-options-settings.feature [hunk 2]
```diff
@@ -14,23 +14,96 @@ Feature: MARC transformation options settings
...
+    * def marc = read(samplesPath + 'marc-transformation-options/create-marc-transformation-options.json')
+    Given path centralServer1.id + '/marc-transformation-options'
+    And request marc
+    When method POST
+    Then status 500
+
+  @get
+  Scenario: Get MARC transformation options settings by central server id
+    * print 'Get MARC transformation options settings by central server id'
+    Given path centralServer1.id + '/marc-transformation-options'
```
Comment (oliinyko) [inline mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/marc-transformation-options-settings.feature:48]: define the path in background and reuse it

**Review by oliinyko — APPROVED** [review]

File: mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/marc-transformation-options-settings.feature (hunk 2 above)
Comment (aleksandr2110) [reply mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/marc-transformation-options-settings.feature]: Done

**Pull request merged**